eslint.config.js

```diff
@@ -1,56 +1,42 @@
-import globals from 'globals';
-import tsPlugin from '@typescript-eslint/eslint-plugin';
-import tsParser from '@typescript-eslint/parser';
-import vuePlugin from 'vue-eslint-parser';
-import eslintJsPlugin from '@eslint/js';
-import stylistic from '@stylistic/eslint-plugin';
-
-const gqlSchemaPath = '././typeDefs/schema.graphql';
+import eslint from '@eslint/js'
+import tseslint from 'typescript-eslint'
+import stylistic from '@stylistic/eslint-plugin'
 
 export default [
-    // GLOBAL configuration
     {
+        // GLOBAL configuration
         ignores: [
             'dist/*',
-            'output/*',
+            '.output/*',
             '.nuxt/*',
             'coverage/*',
             'cypress/videos/*',
-            '.yarn/*'
+            '.yarn/*',
+            'typedefs/gqlTypes.ts'
```

Comment:
we don't want this to be gitignored

Reply:
we want to know when changes are made to the api and this is one of the easiest ways to see. It also makes the build faster when it already exists. (neglible though)